src/ClientProtos/capture_data.proto

```diff
@@ -164,6 +167,36 @@ message TimerInfo {
   uint64 timeline_hash = 11;
   repeated uint64 registers = 12;
   Color color = 13;
+  uint64 group_id = 14;
+  uint64 id = 15;
+  uint64 address_in_function = 16;
+  string name = 17;
+}
+
+message ApiStringEvent {
+  int32 process_id = 1;
+  int32 thread_id = 2;
+  uint64 timestamp_ns = 3;
+
+  uint64 id = 4;
+  string name = 5;
+}
+
+message ApiTrackValue {
```

Comment:
I'm missing some context: how would these be used and why were they not needed until now?

Reply:
They are special types of `ApiEvents`. Currently encoded as timers, even though they only refer to a single point in time (and not a timespan).

`ApiStringEvent` are used to attach a dynamic label to an async. scope.
`ApiTrackValue` events will create a new `ValueTrack` per given name (so name will be the track name). On this track, you can see the value at a specific timestamp as a graph.